pysat/_instrument.py

```diff
@@ -1888,6 +1974,212 @@ def concat_data(self, new_data, prepend=False, **kwargs):
         self.data = concat_func(new_data, **kwargs)
         return
 
+    def custom_attach(self, function, kind='modify', at_pos='end', args=[],
+                      kwargs={}):
+        """Attach a function to custom processing queue.
+
+        Custom functions are applied automatically whenever `.load()`
+        command called.
+
+        Parameters
+        ----------
+        function : string or function object
+            name of function or function object to be added to queue
+        kind : {'add', 'modify', 'pass'}
+            - add
```

Comment:
```suggestion
        kind : str
            Specifies the type of interaction between the custom function and the
            Instrument data.  Accepts one of 'add', 'modify', or 'pass'.
            - add
```

Reply:
Your style may have been fine, but I hadn't seen it before/in awhile so I made a suggestion.